backend/dataall/api/Objects/DataPipeline/resolvers.py

```diff
@@ -252,16 +255,6 @@ def get_stack(context, source: models.DataPipeline, **kwargs):
     )
 
 
```

Comment:
Not needed, since now all targetUri stacks are called with the DataPipelineUri directly